tests/phpunit/tests/blocks/register.php

```diff
@@ -431,6 +431,71 @@ public function test_metadata_not_found_in_the_current_directory() {
 		$this->assertFalse( $result );
 	}
 
+	/**
+	 * Test registering a block using arguments instead of a block.json file.
+	 *
+	 * @ticket 56865
```

Comment:
```suggestion
	 * Tests registering a block using arguments instead of a block.json file.
	 *
	 * @ticket 56865
	 *
	 * @covers ::register_block_type_from_metadata
```